src/coreComponents/physicsSolvers/multiphysics/PoromechanicsSolver.hpp

```diff
@@ -212,6 +212,18 @@ class PoromechanicsSolver : public CoupledSolver< FLOW_SOLVER, MECHANICS_SOLVER
     m_performStressInitialization = performStressInitialization;
   }
 
+  virtual bool updateConfiguration( DomainPartition & domain ) override
+  {
+    if( m_performStressInitialization ) // assume no configuration change during initialization step
```

Comment:
@CusiniM @castelletto1 @Guotong-Ren @jhuang2601 please let me know if this makes sense...